maro/cli/grass/lib/services/master_agent/local_agent.py

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import json
+import multiprocessing as mp
+import os
+import subprocess
+import sys
+import time
+
+import redis
+
+from maro.cli.grass.lib.services.utils.exception import ResourceAllocationFailed
+from maro.cli.grass.lib.services.utils.name_creator import NameCreator
+from maro.cli.grass.lib.services.utils.params import JobStatus
+from maro.cli.utils.cmp import resource_op
+from maro.cli.utils.details_reader import DetailsReader
+
+START_CONTAINER_COMMAND = (
```

Comment:
I am wondering do we need to support Windows? If yes, the command might be created another set for Windows.

Reply:
Yes, i think we should support Windows environment. I am not tested in Windows environment, just follow the design of grass azure which should be tested. Do we need further discussion about this?